src/panel/balloon/contextualballoon.js

```diff
@@ -235,3 +383,85 @@ export default class ContextualBalloon extends Plugin {
 		return position;
 	}
 }
+
+// Rotator view. Used to display last view from the panel stack.
+// Provides navigation buttons for switching panels.
+//
+// @private
+// @extends module:ui/view~View
+class RotatorView extends View {
+	constructor( locale ) {
+		super( locale );
+
+		const bind = this.bindTemplate;
+
+		// Defines whether navigation is visible or not.
+		//
+		// @member {Boolean} #isNavigationVisible
+		this.set( 'isNavigationVisible', true );
+
+		// Navigation button to switches panel to the next one.
+		//
+		// @type {module:ui/button/buttonview~ButtonView}
+		this.buttonPrevView = this._createButtonView( locale.t( 'Prev' ) );
+
+		// Navigation button to switches panel to the previous one.
+		//
+		// @type {module:ui/button/buttonview~ButtonView}
+		this.buttonNextView = this._createButtonView( locale.t( 'Next' ) );
+
+		// Collection of the child views which creates balloon panel contents.
```

Comment:
```suggestion
		// Collection of the child views which creates rotator panel contents.
```
Copy -> Paste -> Read -> Update